src/js/options.js

```diff
@@ -201,11 +205,37 @@ var SettingsTab = {
   initialize: function () {
     fetch("views/settings.html")
       .then((response) => response.text())
-      .then((htmlData) => {
+      .then(async (htmlData) => {
         // Insert template file.
         document.querySelector('main section[data-tab="settings"]').innerHTML =
           htmlData;
 
+        await SettingsTab.injectData(CONFIG_FILE_OPTIONS, "config");
+        await SettingsTab.injectData(SEARCH_RESULT_OPTIONS, "search-results");
+        await SettingsTab.injectData(MERGE_OPTIONS, "merge-options");
+
+        // Initialize tooltips.
+        let popovers = document.querySelectorAll("[data-title]");
```

Comment:
This selector is to _broad_: it's targeting all elements on the page that have `data-title` (right now we only have tooltips here, but this will cause troubles if we add tooltips or popovers elsewhere), the selector should be only affecting elements on the settings tab. something like `document.querySelectorAll('main section[data-tab="settings"] [data-title]')`